### PR TITLE
qm-monitor: fix filename

### PR DESCRIPTION
**Pull request by saw-jan**
Refers #8537

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/qm-monitor.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# qm monitor
+
+> Enter Qemu Monitor interface.
```
Comment (kbdharun) [inline pages/linux/qm-monitor.md:3]:
```suggestion
> Enter the QEMU Monitor interface.
```

File: pages/linux/qm-monitor.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Enter Qemu Monitor interface of a specific virtual machine:
```
Comment (kbdharun) [inline pages/linux/qm-monitor.md:6]:
```suggestion
- Enter the QEMU Monitor interface of a specific virtual machine:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

> Edit: I noticed some discrepancies in `qm` pages some use `QEMU` and others use `qemu`. I think we need to update it to a unified one.

**Review by kbdharun — APPROVED** [review]

**Comment by navarroaxel** [comment]
Oh! we made a mistake, could you merge with `main` to resolve the Git conflicts and fix this PR #8828 with your page? Thanks!
The merged PR has an incorrect filename.

**Comment by navarroaxel** [comment]
@saw-jan Oh! we made a mistake, could you merge with main to resolve the Git conflicts and fix this PR https://github.com/tldr-pages/tldr/pull/8828 with your page? Thanks!
The merged PR has an incorrect filename.

**Comment by kbdharun** [comment]
> @saw-jan Oh! we made a mistake, could you merge with main to resolve the Git conflicts and fix this PR #8828 with your page? Thanks! The merged PR has an incorrect filename.

Sorry, I noticed this just now. @saw-jan can you merge your branch with the`main` and remove the other page with the incorrect name (Location: pages/linux/qmmonitor.md) in this PR?

**Comment by saw-jan** [comment]
@navarroaxel @kbdharun I have made the requested changes.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**